docs/02_getting-started/03_smart-contract-development/11_create-tic-tac-toe-contract.md

```diff
@@ -0,0 +1,709 @@
+---
+content_title: "2.10 Creating a Tic-Tac-Toe Smart Contract"
+link_text: "2.10 Creating a Tic-Tac-Toe Smart Contract"
+---
+
+## Goal
+
+The following tutorial will guide you to build a sample Player vs Player game contract. We will apply the knowledge we acquired eariler and use the tic tac toe game to demonstrate.
+
+## The rules of the game
```

Comment:
For all our titles and headings, we can use Title-style Capitalization. Meaning, to capitalize all the words in th heading except for a, an, the, and other prepositions 

Reference: https://docs.microsoft.com/en-gb/style-guide/capitalization#title-style-capitalization

Suggestion:

# The Rules of the Game

Reply:
Looks Good

Reply:
> For all our titles and headings, we can use Title-style Capitalization. Meaning, to capitalize all the words in th heading except for a, an, the, and other prepositions
> 
> Reference: https://docs.microsoft.com/en-gb/style-guide/capitalization#title-style-capitalization
> 
> Suggestion:
> 
> ## The Rules of the Game

we do not have this rule/convention applied everywhere, I'd say our all documentation is 50% using this convention right now.